app/ui/archiveTile.js

```diff
@@ -117,8 +117,8 @@ function password(state, emit) {
     return html`
       <input
         id="${inputId}"
-        class="border rounded focus:border-blue-60 leading-normal my-1 py-1 px-2 h-8 dark:bg-grey-100"
-        style="width: 100%"
+        class="border rounded focus:border-blue-60 leading-normal my-1 py-1 px-2 h-8 dark:bg-grey-100 w-full"
+        style="max-width: 23rem;"
```

Comment:
reason for this: make sure both boxes are both the same size, since they can take up the rest of the room to the right of the 'flex-grow'